src/seqan.cc

```diff
@@ -62,6 +62,9 @@ void seqan_startup() {
   
   sa.def("atEnd<SeqFileIn>"_raw,+[](SeqFileIn& sf) {return atEnd(sf); } );
 
+  sa.def("position(SeqFileOut&)"_raw,+[](SeqFileOut& sf) { return (int)position(sf); });
```

Comment:
Can you change these to return size_t?  int won't be big enough.

Reply:
Done.